styling updates

braedonwatkins opened the pull request:
changed sizing, color palette, and cleaned up redundant styling

braedonwatkins committed:
basic color palette

braedonwatkins committed:
login page & components color updates

braedonwatkins committed:
index page color update

braedonwatkins committed:
about page color update

braedonwatkins committed:
logo color update

SVG didn't wanna cooperate but that's a problem for future me

braedonwatkins committed:
navbar color update

braedonwatkins committed:
forgotpassword color update

braedonwatkins committed:
filezone color update

braedonwatkins committed:
navbar backdrop, formatting, & hover

braedonwatkins committed:
wider palette

braedonwatkins committed:
apply wider palette

braedonwatkins committed:
hover override fix

braedonwatkins committed:
better nav hover bg contrast

braedonwatkins committed:
"correct gap div to right side"

braedonwatkins committed:
global color application

braedonwatkins committed:
discard duplicate height & width fills

braedonwatkins committed:
left navbar gap

braedonwatkins committed:
fix fill change width for OAuth

xTye commented on src/components/Filezone.tsx:
```diff
@@ -1,12 +1,12 @@
 export default function Example() {
```
This should not be default, and the naming convention is off. The correct way to format is `export const Filezone = () => {}`

braedonwatkins replied:
gotchya. since this is how it looks on develop, and this branch only addresses styling, i'm just going to merge and i can add a quick branch to address this syntax for better organization.

xTye commented on src/components/Register.tsx:
```diff
@@ -8,53 +8,53 @@ export const Register = (props: RegisterProps) => {
     <>
       <form action="/api/register" method="POST" className="flex flex-col w-64 gap-6">
-        <div className="text-3xl">
+        <div className="text-3xl ">
```
fix space here

xTye commented on src/pages/about.tsx:
```diff
@@ -4,11 +4,11 @@ import { Navbar } from '../components/Navbar'
   return (
     <>
-      <div className="w-full h-full bg-amber-50">
+      <div className=" ">
```
delete className if empty

braedonwatkins replied:
good catch thanks

xTye commented on src/pages/index.tsx:
```diff
@@ -5,26 +5,26 @@ import { Logo } from '../components/Logo'
   return (
     <>
-      <div className="w-full h-full bg-amber-50">
+      <div className=" h-full">
```
delete space in front

xTye commented on src/pages/index.tsx:
```diff
@@ -5,26 +5,26 @@ import { Logo } from '../components/Logo'
           page="home"
         />
-        <div className="flex flex-col w-full px-28 mt-10 text-green-800">
+        <div className="flex flex-col  px-28 mt-10 ">
```
delete double space here and end space

xTye commented on src/pages/index.tsx:
```diff
@@ -5,26 +5,26 @@ import { Logo } from '../components/Logo'
             Welcome to EMURR
           </div>
-          <div className="flex justify-between w-full mt-5">
+          <div className="flex justify-between  mt-5">
```
delete double space

xTye commented on src/pages/index.tsx:
```diff
@@ -5,26 +5,26 @@ import { Logo } from '../components/Logo'
             </div>
           </div>
-          <div className="text-5xl mt-40 font-light">
+          <div className="text-5xl mt-40 font-light ">
```
delete end space

xTye commented on src/pages/index.tsx:
```diff
@@ -5,26 +5,26 @@ import { Logo } from '../components/Logo'
             Meet the team
           </div>
-          <div className="flex flex-col justify-between w-full mt-5">
+          <div className="flex flex-col justify-between  mt-5 ">
```
delete double space and end space

xTye commented on src/pages/login.tsx:
```diff
@@ -25,16 +25,16 @@ const LoginPage: NextPage = () => {
   return (
     <>
-      <div className="w-full min-h-screen bg-amber-50">
+      <div className=" min-h-screen ">
```
delete spaces before and after

xTye commented on src/pages/tinymce.tsx:
```diff
@@ -21,8 +21,8 @@ const TinyMCEPage: NextPage = () => {
-      <div className="w-full h-full flex flex-col justify-center items-center">
-        <div className="flex w-full justify-center items-center">
+      <div className="  flex flex-col justify-center items-center">
+        <div className="flex  justify-center items-center">
```
this line and the one above has two spaces before and a double space

xTye commented on src/pages/tinymce.tsx:
```diff
@@ -43,12 +43,12 @@ const TinyMCEPage: NextPage = () => {
           />
           </div>
-          <div className='prose lg:prose-xl prose-zinc h-full w-1/2 ml-5'>
+          <div className='prose lg:prose-xl prose-zinc  w-1/2 ml-5'>
```
double space here

xTye reviewed (approved):
Overall looks good. The spacing is a little off, but I think that has to do with deletions. Overall, nice work! I'm going to approve. Once the changes are implemented, and Omar approves, feel free to merge.

braedonwatkins committed:
code formatting reqs

braedonwatkins committed:
missed 2xl login

omosa commented on src/components/Navbar.tsx:
```diff
@@ -7,30 +7,30 @@ export interface NavbarProps {
-      <nav className="sticky top-0 z-50 w-full text-stone-500 border-b border-gray-200 bg-amber-50">
-        <div className="flex justify-between h-auto w-full py-5 px-10">
-          <div className="flex flex-row justify-center items-center">
+      <nav className="sticky top-0 z-50 border-b border-brown backdrop-blur-md">
```
```suggestion
      <nav className="sticky top-0 z-50 border-b border-brown backdrop-blur-md bg-background-400">
```
Navbar currently has a transparent background. Text shows up underneath it when scrolling down a page. Solve this by adding a background.

omosa replied:
Update after discussing this with Braedon: I had not realized there was a backdrop blur for the navbar; this feature was working in Chrome and Edge, but not for Firefox until I updated to a more recent version (I was running v99). See https://bugzilla.mozilla.org/show_bug.cgi?id=1578503 which was only fixed in July (Firefox v103).

braedonwatkins committed:
revert Navbar blur

Navbar blur is only supported in Firefox 103.* which was released in July 2022.
This is too recent to risk a huge loss of readability just for a styling change.
Reverting to a solid background color for now.
